recipes/libarchive/all/conandata.yml

```
@@ -25,6 +25,9 @@ sources:
     sha256: "8643d50ed40c759f5412a3af4e353cffbce4fdf3b5cf321cb72cacf06b2d825e"
 patches:
   "3.7.1":
+    - patch_file: "patches/0007-3.7.1-include-getopt.patch"
+      patch_description: "include getopt.h if HAVE_GETOPT_OPTRESET is defined"
+      patch_type: "portability"
```

@klausholstjacobsen Thanks for making a PR upstream! If you could add a `patch_source` field like below, that will help us keep track of when the patch has been upstreamed.

```suggestion
      patch_type: "portability"
      patch_source: "https://github.com/libarchive/libarchive/pull/2007"
```